### PR TITLE
Document multi node connectivity troubleshooting

### PR DESCRIPTION
**Pull request by tgraf**

### CODE REVIEW

File: Documentation/troubleshooting.rst [hunk 1]
```diff
@@ -16,7 +16,46 @@ We use GitHub issues to maintain a list of `Cilium Frequently Asked Questions
...
 addressed.
 
+Connectivity Issues
+===================
+
+Node to node traffic is being dropped
+-------------------------------------
+
+Sympton
+~~~~~~~
```
Comment (aalemayhu) [inline Documentation/troubleshooting.rst]: s/Sympton/Symptom ?

File: Documentation/troubleshooting.rst [hunk 2]
```diff
@@ -16,7 +16,46 @@ We use GitHub issues to maintain a list of `Cilium Frequently Asked Questions
...
+1. Run `cilium monitor` on the node of the source and destination endpoint.
+   Look for packet drops.
+
+When running in :ref:`arch_overlay` mode:
+
+2. Run `cilium bpf tunnel list` and verify that each Cilium node is aware of
+   the other nodes in the cluster.  If not, check the logfile for errors.
+
+3. If nodes are being populated corectly, run `tcpdump -n -i cilium_vxlan` on
+   each node to verify whether cross node traffic is being forwarded correctly
```
Comment (aalemayhu) [inline Documentation/troubleshooting.rst]: s/corectly/correctly/g

**Review by aalemayhu — APPROVED** [review]
Minor typos, but LGTM.

**Review by aanm — APPROVED** [review]

File: Documentation/troubleshooting.rst (hunk 1 above)
Comment (tgraf) [reply Documentation/troubleshooting.rst]: fixed

File: Documentation/troubleshooting.rst (hunk 2 above)
Comment (tgraf) [reply Documentation/troubleshooting.rst]: fixed

**Pull request merged**